include/sys/abd.h

```diff
@@ -177,19 +177,20 @@ abd_zero(abd_t *abd, size_t size)
 static inline boolean_t
 abd_is_linear(abd_t *abd)
 {
-	return ((abd->abd_flags & ABD_FLAG_LINEAR) != 0);
+	return ((abd->abd_flags & ABD_FLAG_LINEAR) != 0 ? B_TRUE : B_FALSE);
```

Comment:
```suggestion
	return ((abd->abd_flags & ABD_FLAG_LINEAR) ? B_TRUE : B_FALSE);
```

That's kind of unfortunate.  If we must return `B_TRUE` / `B_FALSE` I'd suggest dropping the `!= 0` bit which is redundant and I don't think helps readability in this case.

Reply:
Done 👍